README.md

```diff
@@ -124,28 +124,32 @@ export function detect(source) {
   return source.match(/some-test/i) !== null;
 }
 
-export function parse({source, generateSourceMap}, done) {
-  // Here you convert the source into refract elements.
+export function parse({minim, generateSourceMap, source}, done) {
```

Comment:
I meant, we should have the line `import minim` at the beginning of this code example ```` ```js ```` so that people won't wonder what `minim` is in this line.